certified-connectors/Desk365/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1406 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Desk365",
+    "description": "Desk365 is a cloud-based modern helpdesk for the Microsoft 365 workplace that lets you deliver outstanding customer service through channels like Microsoft Teams, Email, Web Forms/Widgets and more. Automate repetitive work and save time with Desk365’s intuitive, feature-rich web app that comes with a unified inbox to manage all your customer conversations.",
+    "version": "1.0",
+    "contact": {
+      "name": "Desk365 Support",
+      "url": "https://www.desk365.io/contact/",
+      "email": "help@desk365.io"
+    }
+  },
+  "host": "apps.desk365.io",
+  "basePath": "/apis",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/": {
+      "x-ms-notification-content": {}
```

Comment:
@Desk365-vdpk can you please explain this empty operation? is it possible to remove it?